src/app/containers/BulletedListItem/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import Blocks from '../Blocks';
+import paragraph from '../Paragraph';
+import { listItemPropTypes } from '#models/propTypes/list';
+
+const componentsToRender = { paragraph };
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const BulletedListItemContainer = ({ blocks }) => {
+  return (
+    <li>
+      <Wrapper>
```

Comment:
Is Wrapper necessary?  could the styles be applied to the parent `li`?

Reply:
Wrapper is necessary to align the list item content to the bullet by putting it in an`inline-block`. Since the div is required for stylingI left the styles in the div.

Reply:
Sounds dubious because I thought the bullet was a background image meaning the position of it can be controlled with background-position

Reply:
The bullet is a background image of the before pseudo element. It was important to style the element in an inline-block to achieve the effect of having a multi-line component stay to the proper side of the bullet

Reply:
Because the paragraph grid is a block element, this caused the paragraph to come below the bullet. Wrapping it in an inline-block solves the issue.

Reply:
Oh ok. I wouldn't have expected the use of a psuedo element. Did you consider trying this?

```css
ul {
  list-style-image: url(‘images/sample-bullet.png’);
}
```

from https://dev.to/neshaz/css-bullet-style-explained-3lm5

Reply:
I believe if there's a visibile bullet using `list-style`, screen readers will read out "bullet".

One of Catherine's requirements for this was that screen readers do not do that, hence the `list-style` of `none`, and then adding a "fake" bullet with a pseudo-element.

Reply:
sounds like a nice idea for a test e.g. 'it should not use default bullet styling so that screen readers don't read out "bullet"' this would also prevent devs refactoring and breaking this requirement

Reply:
That sounds like a test for the psammead component itself. I'll raise fix that in a psammead PR.

Reply:
https://github.com/bbc/psammead/pull/2698